### PR TITLE
Update README to include scope, examples, process

### PR DESCRIPTION
**Pull request by heather999**
Addresses #9

### CODE REVIEW

**Review by johannct — APPROVED** [review]
lgtm. Perhaps "decision's implementation" reads bad as decision can be taken *not* to implement.... I leave you find something syntactically working :)

**Comment by heather999** [comment]
Ok - trying to make it clearer, how about:
"adopt a course of action, and track implementation." maybe that could be "and track any implementation"

File: README.md [hunk 1]
```diff
@@ -1,30 +1,45 @@
...
 Before an RFC may be adopted, one or more GitHub issues should be created to capture the work required to implement the RFC’s decision.
 
 * Label the RFC issue as *adopted* 
 * Add the RFC issue to the [*Adopted* Project](https://github.com/LSSTDESC/RequestForComments/projects/1)
 
-**Withdrawing an RFC**
+## Withdrawing an RFC
 
-If the RFC cannot be adopted, then it can be withdrawn. Assign the *withdraw* label to the RFC issue.
+If the RFC cannot be adopted, then it can be withdrawn. Assign the *withdraw* label to the RFC issue and close.
```
Comment (eacharles) [inline README.md]: close -> closed

**Comment by heather999** [comment]
Thanks all for taking a look!  I'm going to merge this and move on to the draft announcement.

**Pull request merged**